cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -419,3 +421,56 @@ def test_variable_pass(request, appliance, setup_ansible_repository, import_data
         provision_request = appliance.collections.requests.instantiate(request_description)
         provision_request.wait_for_request(method="ui")
         request.addfinalizer(provision_request.remove_request)
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1677575])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1677575.zip", "bz_1677575", None)],
+    ids=["datastore"],
+)
+def test_import_domain_containing_playbook_method(request, appliance, setup_ansible_repository,
+                                                  import_data):
+    """This test case tests support of Export/Import of Domain with Ansible Method
+
+    Bugzilla:
+        1677575
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.11
+        casecomponent: Automate
+        tags: automate
+        setup:
+            1. Add playbook repository
+            2. Create a new automate method with playbook type.
+            3. Fill the required fields, for instance repository and playbook.
+            4. Export this datastore.
+            5. Playbook method fields are stored as a names instead of IDs. (this is not
+               possible via automate need to check manually in method yaml)
+        testSteps:
+            1. Import the exported datastore.
+            2. Change name of playbook in method yaml to invalid playbook name
```

Comment:
Where is this happening in the test ?

Reply:
@sshveta 
We cant do those things with automation. I mean, we can not export datastore. Hence I did those things manually and importig that datastore.
Updated test steps with note:
```
testSteps:
            1. Import the exported datastore and change name of playbook in method yaml to invalid
               playbook name(Note: These test steps needs to execute manually and then import
               datastore)
        expectedResults:
            1. Proper error should be displayed while importing datastore with invalid playbook
```